R/upset.R

```diff
@@ -21,7 +21,16 @@
 #'      TRUE ~ "Other"
 #'     )
 #'   )
-#' crash_summary = tc_join_stats19(crashes_wf, casualties_wf2, vehicles_wf)
+#' table(vehicles_wf$vehicle_type)
+#' vehicles_wf2 = dplyr::mutate(
+#'   vehicles_wf,
+#'   vehicle_type_simple = dplyr::case_when(
+#'      vehicle_type == "Car" ~ "Car",
+#'      vehicle_type == "Cyclist" ~ "Pedal cycle",
```

Comment:
Good question and well spotted. I think for casualties it's because of this line:

https://github.com/saferactive/trafficalmr/blob/bce0785abcb3df34a9e16d3594563b5f28a82536/R/upset.R#L122

I don't see an equivalent for "Pedal cycle" so have updated the example accordingly. Great fix!